Update How-to-Monitor-Docker-Containers.md (#55)
louislam/uptime-kuma-wiki: How-to-Monitor-Docker-Containers.md

My original ExecStart was: `ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock`, note the -H that would cause a duplicate property error.

> ### Method 2 - If you installed docker using snap

Reviewer: Method 2 is mentioned above. Is this heading intentional?

Do you think an edendum using a `Note` (see https://github.com/orgs/community/discussions/16925) would be better?

Author: Didn't know about the "note" it makes perfect sense to use it

Reviewer: No, I meant if this might be more readable if the part, that snap stores this file somewere else, should be up where we tell the default path and added as a `Note`

Reviewer: Doing so would also remove the need to have a separate section on how the file needs to be modified

> Tested in Ubuntu 22
> 

Reviewer: The rest of the doc also does not have such statements
```suggestion
```

> The location of the files is different:  **daemon.json** is in /var/snap/docker/current/config/

Reviewer: ```suggestion
Snap stores the `daemon.json` here:  `/var/snap/docker/current/config/daemon.json`
```

> `sudo nano /var/snap/docker/current/config/daemon.json`

Reviewer: ```suggestion
use `sudo nano /var/snap/docker/current/config/daemon.json` to edit the file like
```

> And this is what the original file looks like
> 
> ```
> {
>     "log-level":        "error",
>     "storage-driver":   "overlay2"
> }
> ```
> 
> Change it to 
> 
> ```
> {
>     "log-level":        "error",
>     "storage-driver":   "overlay2",
>     "hosts": ["unix:///var/run/docker.sock", "tcp://0.0.0.0:2375"]
> }

Reviewer: see https://stackoverflow.com/questions/40883421/diff-syntax-highlighting-in-github-markdown
```suggestion
```diff
{
    "log-level":        "error",
    "storage-driver":   "overlay2",
+   "hosts": ["unix:///var/run/docker.sock", "tcp://0.0.0.0:2375"]
}

```

> - Restart the service: 
> `sudo systemctl restart snap.docker.dockerd.service`
> 
> - Check if the service is running:
> `sudo systemctl status snap.docker.dockerd.service`
> 
> The service should be running as usual, showing the docker snap service
> 
> ![RustDesk - 09-01 at 22 18 35@2x](https://github.com/louislam/uptime-kuma/assets/642149/8494c876-5580-4f87-9ceb-9a5974f1c977)

Reviewer: I would prefer either

```suggestion
- Restart the service using `sudo systemctl restart snap.docker.dockerd.service`
- Check if the service is running using `sudo systemctl status snap.docker.dockerd.service`

The service should be running as usual, showing the docker snap service

![Screenshot showing the snap docker service working](https://github.com/louislam/uptime-kuma/assets/642149/8494c876-5580-4f87-9ceb-9a5974f1c977)
```

Reviewer: or
```suggestion
- Restart the service: 
  ```bash
  sudo systemctl restart snap.docker.dockerd.service
  ```
- Check if the service is running:
  ```bash
  sudo systemctl status snap.docker.dockerd.service
  ```

The service should be running as usual, showing the docker snap service

![Screenshot showing the snap docker service working](https://github.com/louislam/uptime-kuma/assets/642149/8494c876-5580-4f87-9ceb-9a5974f1c977)
```

> 
> 

Reviewer: ```suggestion
```
